.github/workflows/_bump-version.yml

```diff
@@ -0,0 +1,80 @@
+name: "Bump version"
+
+on:
+    workflow_call:
+        inputs:
+            package:
+                description: "Choose the package to bump"
+                type: string
+                required: true
+            branch:
+                description: "Choose the branch to use"
+                type: string
+                default: "main"
+            version:
+                description: "Choose the version to bump to (e.g. 1.2.3rc1, patch, release)"
+                type: string
+                default: "beta"
+        secrets:
+            FISHTOWN_BOT_PAT:
+                description: "Token to commit/merge changes into branches"
+                required: true
+    workflow_dispatch:
+        inputs:
+            package:
+                description: "Choose the package to bump"
+                type: choice
+                options:
+                -   "dbt-adapters"
+                -   "dbt-tests-adapter"
+                -   "dbt-athena"
+                -   "dbt-bigquery"
+                -   "dbt-postgres"
+                -   "dbt-redshift"
+                -   "dbt-snowflake"
+                -   "dbt-spark"
+            branch:
+                description: "Choose the branch to use"
+                type: string
+                default: "main"
+            version:
+                description: "Choose the version to bump to (e.g. 1.2.3rc1, patch, release)"
+                type: string
+                default: "beta"
+        secrets:
+            FISHTOWN_BOT_PAT:
+                description: "Token to commit/merge changes into branches"
+                required: true
```

Comment:
You don't need this defined for the workflow_dispatch trigger.  Wither this trigger all secrets available in the context are just available.  They just need to be defined for workflow_call.